TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8260,7 +8260,8 @@ execStatementText: ExecStatementText;
 execStatementEnd: END_EXEC;
 
 //FOR SQL
-commitStatement: SQL_COMMIT;
+// Defining 'WORK' as contextual keyword here since it is not part of reserved words
```

Comment:
Yes, it can even be improved a bit. see my remark in #2209 about ddefining it as a re-usable contextual keyword.